e2e/Makefile

```diff
@@ -1,20 +1,25 @@
 export GO111MODULE=on
 
-IMAGE_VERSION=canary
-K8S_VERSION?=""
-GINKGO_PATH := $(shell go env GOPATH)/bin/ginkgo
+IMAGE_VERSION?=canary
+K8S_VERSION?="v1.28.7"
 
-.PHONY: test clean check-token install-terraform test-manifest
+.PHONY: test clean check-token test-manifest
 
-test-manifest:
+test-manifest: kustomize
 	../hack/generate-yaml.sh $(IMAGE_VERSION) > test/manifest/linode-blockstorage-csi-driver.yaml
 
-${GINKGO_PATH}:
-	go install github.com/onsi/ginkgo/v2/ginkgo@v2.13.2
+test-cluster: ctlptl clean-cluster
+	./test/scripts/create_management_cluster.sh $(K8S_VERSION)
 
-test: ${GINKGO_PATH} check-token test-manifest
+clean-cluster: ctlptl
+	ctlptl delete cluster kind-management --ignore-not-found
+	rm -rf ./test/kind-management.conf
+
+test: clusterctl ginkgo check-token test-manifest test-cluster
 	go list -m; \
-	ginkgo -r --vv --procs=6 --trace --cover $(TEST_ARGS) -- --v=3 --k8s-version=${K8S_VERSION} $(SUITE_ARGS)
+	ginkgo -r --vv --procs=1 --trace --cover $(TEST_ARGS) -- --v=3 --k8s-version=${K8S_VERSION} $(SUITE_ARGS)
```

Comment:
There is a limitation on Linode API, that's why only `procs=1` supported at the moment.

Reply:
I've been running tests today with procs=2 and it takes 50% less time and hasn't showed any issues. wdyt?

Reply:
@avestuk could you please change it on your polishing PR?

Reply:
Yes I already have :D